resources/cmake_config.cmake.in

```diff
@@ -0,0 +1,16 @@
+# - Config file for the FooBar package
```

Comment:
What is this file used for?

Reply:
nothing I will remove it ^^
I wanted to provide a way to have a default `${PROJECT_NAME}Config.cmake`
But it seem rather annoying, and you do not gain so much doing it.